src/fragments/lib/storage/js/browser-uploads.mdx

```diff
@@ -0,0 +1,24 @@
+Upload an image in the browser:
```

Comment:
```suggestion
## Browser uploads

Upload an image in the browser:
```

Reply:
For JS, we need to keep title to `Browser uploads` because the code example is for browser uploads. For RN only, we need to update it to `Media Uploads`.